pkg/admin/facade/resoucescope.go

```diff
@@ -62,3 +65,25 @@ func (f *ResourceScopeFacade) GetScope(ctx context.Context, id string) (*model.S
 func (f *ResourceScopeFacade) ListScopes(ctx context.Context, resourceID string) ([]*model.Scope, error) {
 	return f.ResourceScopeQueries.ListScopes(ctx, resourceID)
 }
+
+func (f *ResourceScopeFacade) ListResources(ctx context.Context, options *resourcescope.ListResourcesOptions, pageArgs graphqlutil.PageArgs) ([]model.PageItemRef, *graphqlutil.PageResult, error) {
```

Comment:
Again, I forgot to specify the order of the list. The list should be ordered by created_at ASC. Add an index.

Reply:
I've added ordering in https://github.com/authgear/authgear-server/pull/5278/files#diff-ef4ddd48e9637013f387a224ca40b2a7f31c0ef6cedda79eeb351c23739d7b51R179 by `uri`.